docs/source/model_library.rst

```diff
@@ -144,13 +144,34 @@ performed on the association metadata).
    any "borrows" using the indices do. Be sure to open the library before
    trying to borrow a model.
 
+
+.. _library_association_information:
+
+Association Information
+=======================
+
 `~stpipe.library.AbstractModelLibrary.asn` provides read-only access to the association data.
 
 .. code-block:: pycon
 
    >>> library.asn["products"][0]["name"]
    >>> library.asn["table_name"]
 
+Although the specifics of what is returned by `~stpipe.library.AbstractModelLibrary.asn`
+depends on how the subclass implements ``AbstractModelLibrary._load_asn``, it
+is required that the association metadata dictionary contain a "members" list. This
+can be inspected via ``library.asn["products"][0]["members"]`` and must contain a
```

Comment:
what does the index `[0]` correspond to?

Reply:
The 0th index (only) product since "products" is a list: https://jwst-pipeline.readthedocs.io/en/latest/jwst/associations/level3_asn_technical.html#example-association